docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,85 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+4+| Create operations
 
 |`* * *` |user |add a new person |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |add tags to a person |categorise the people I know
+
+|`* * *` |user |add relationships between people |show that two persons know each other
+
+|`* *` |user |add link:#named-relationship[named relationships] between people |show that two persons know each other via a specific method
+
+|`* *` |user |add link:#confidence-estimate[confidence estimate] to a person's attributes |differentiate between verified and unverified pieces of information
+
+|`* *` |user |add remarks about a relationship between people |record detailed information about the relationship between two people
+
+|`* *` |user with many persons in the address book |add profile pictures to each person |distinguish between people with the same name
+
+4+| Read operations
+
+|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+
+|`* * *` |user |see the people a person is related to |find people related to a person
+
+|`* * *` |user |see the history of commands I have typed |keep track of what I have done to my address book
```

Comment:
this is not a must have feature; 3 star high priority features are those that are essential for a product; that is what would make it a minimum viable product. reconsider priorities of user stories.